scripts/oneoff/acknowledge_publish_draft_service_updates.py

```diff
@@ -0,0 +1,37 @@
+#!/usr/bin/env python
+
+import argparse
+import multiprocessing as mp
+from multiprocessing.pool import ThreadPool
+import os
+import sys
+
+from dmapiclient.audit import AuditTypes
+from dmapiclient.data import DataAPIClient
+
+sys.path.insert(0, '.')  # noqa
+
+from dmscripts.helpers.auth_helpers import get_api_url, get_auth_token
+
+def acknowledge_update(audit_event):
+    response = data.acknowledge_audit_event(audit_event['id'], user='(dm developers) scripts/publish-g-cloud-draft-services.py')['auditEvents']
```

Comment:
maybe user should be `scripts/acknowledge_publish_draft_service_updates.py`?

Reply:
Yeah sure